src/Sql/Sql/AdvancedThreatProtection/Cmdlet/EnableAzureSqlServerAdvancedDataSecurity.cs

```diff
@@ -77,7 +78,7 @@ protected override ServerAdvancedDataSecurityPolicyModel PersistChanges(ServerAd
                 }
                 else
                 {
-                    ModelAdapter.SetServerAdvancedDataSecurity(model);
+                    ModelAdapter.SetServerAdvancedDataSecurity(model, DefaultContext.Environment.GetEndpoint(AzureEnvironment.Endpoint.StorageEndpointSuffix));
```

Comment:
Why not add `DefaultContext.Environment.GetEndpoint(AzureEnvironment.Endpoint.StorageEndpointSuffix)` to SqlAdvancedDataSecurityAdapter.cs and then you can remove the extra parameter from `SetManagedInstanceAdvancedDataSecurity`.

Like this:
`SqlThreatDetectionAdapter.SetManagedInstanceThreatDetectionPolicy(threatDetectionPolicy, DefaultContext.Environment.GetEndpoint(AzureEnvironment.Endpoint.StorageEndpointSuffix));`

Then you just have to change one file